src/python/gem5/utils/multisim/multisim.py

```diff
@@ -114,6 +115,7 @@ def get_simulator_ids(config_module_path: Path) -> Set[str]:
     )
     p.start()
     p.join()
+    print(dict_return)
```

Comment:
Ditto

Reply:
yeah, I should have this marked as draft. I still like these debug prints there.